OpenRA.Mods.Common/Projectiles/AreaBeam.cs

```diff
@@ -47,9 +49,11 @@ public class AreaBeamInfo : IProjectileInfo
 		[Desc("Ranges at which each Falloff step is defined.")]
 		public readonly WDist[] Range = { WDist.Zero, new WDist(int.MaxValue) };
 
-		[Desc("Maximum offset at the maximum range.")]
```

Comment:
Can we change this to refer to `InaccuracyType` instead of removing it? Likewise in the other projectiles.